src/docs/asciidoc/reference_doc/card_notification.adoc

```diff
@@ -164,38 +164,6 @@ rRule : {
 }
 ....
 
-==== Using `recurrence` field in the timespan object (deprecated)
-
```

Comment:
There is some more modifications to be done above in this documentation : 

- Remove reference of two way of defining recurrence 
- Remove reference to "recurence object"

Reply:
Done.